builder/comp-builder.nix

```diff
@@ -22,16 +22,17 @@
 
 , doCheck ? component.doCheck || haskellLib.isTest componentId
 , doCrossCheck ? component.doCrossCheck || false
-, dontPatchELF ? true
-, dontStrip ? true
+, dontPatchELF ? component.dontPatchELF
+, dontStrip ? component.dontStrip
```

Comment:
Could we keep all the options the same polarity? As a user, I'd find it a bit strange to see `doCheck` and `dontStrip` next to each other. I won't complain about `do` vs `enable` ;)

Reply:
I had the same thought, and in principle I agree. But `stdenv.mkDerivation` uses `dontPatchELF` and `dontStrip` (https://nixos.org/nixpkgs/manual/#ssec-fixup-phase), so I figured I'd be consistent with those.

Reply:
I very much feel the same as @michaelpj; but do agree that we should just stick with the "defaults" for now, even though the defaults seem wrong.

Reply:
I believe the reason for that is that it provides a rough guide to the "defaults": if the option is `dontPatchELF` then the default is to do it. However, we're wrapping these in module options where the default is explicit, so I don't think that applies to us.

Reply:
@michaelpj by "default" I merely meant the "default" of nixpkgs. That is familiarity. I could have chosen my words better.

I'm for keeping them aligned with nixpkgs for easier migration. I am open to a PR that adds the `do`/`dont` variants to all keys though. Eventually we might have deprecated trace marks on them?

Reply:
See #346